memspecs/SAMSUNG_M471B5674QH0_2GB_DDR3-800_64bit_Leakage.xml

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE memspec SYSTEM "memspec.dtd">
```

Comment:
The name of the file should be:

memspecs/SAMSUNG_M471B5674QH0_2GB_DDR3-800_64bit_Measured.xml